Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -11,12 +11,16 @@ include("utils.functions.filesystem.files");
 */
 Wine.prototype.DXVK = function (dxvkVersion) {
     var operatingSystemFetcher = Bean("operatingSystemFetcher");
+    var uiQuestionFactory = Bean("uiQuestionFactory");
     print("NOTE: wine version should be greater or equal to 3.10");
-    if (operatingSystemFetcher.fetchCurrentOperationSystem() != "Linux")
+    if (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() !== "Linux")
     {
-        this.wizard().message(tr("DXVK might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement"));
+        uiQuestionFactory.create(
+				tr("DXVK is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Do you want to continue? Chosing yes will skip DXVK verb  installation and continue with other verbs. Chosing no will quit script installation.", winePrefix),
+				() => return this;
+			);
     }
-    else
+    else (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() == "Linux")
```

Comment:
Shouldn't there be an `if`. Also: use `===`.

Reply:
No I dont think there should be. Also why is double equals not enough?

Reply:
If there should not be an `if`, I do not understand the code.

Reply:
Basically that part after `else` was a leftover from when I wanted this script to behave like @ImperatorS79 suggested.